drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -21,6 +21,16 @@
                       <span class="sr">{{ help_texts.show_title }}</span>
                     </label>
 
+                    <label class="h3" for="problem-mode" title="{{ help_texts.mode }}">{% trans "Problem mode" %}</label>
```

Comment:
Why isn't the display name as defined on the xblock field used instead of hardcoding/duplicating the display name?